mingw-w64-zed/PKGBUILD

```
@@ -113,6 +115,10 @@ build() {
   export CFLAGS="$CFLAGS -I$(cygpath -am ${srcdir}/aws-lc-rs/aws-lc-sys/aws-lc/include)"
 
   cargo build --release --frozen -p zed -p cli
+
+  msg "Build documentation"
+  cd "${srcdir}/${_realname}/docs"
+  "${MINGW_PREFIX}/bin/mdbook.exe" build
```

```suggestion
  mdbook build
```
it's not required to hardcode executable path as `${MINGW_PREFIX}/bin` is first in `$PATH`